File: programming files/datetime_convert.py
#BM
from datetime import datetime

# Date string
date_str = "2022-03-17 10:45:30"
patch-3
# Date string to date object
date_obj = datetime.strptime(date_str, '%m/%d/%Y %H:%M:%S')
# Formatted date object to a reformatted string
formatted_date = date_obj.strftime('%Y-%m-%d %H:%M:%S')

# Prints formatted date
main
print(formatted_date)

File: programming files/datetime_convert.cc
#include <iostream>
#include <iomanip>
#include <sstream>
#include <chrono>

int main()
{
    //initial date string
    std::string date_str = "2022-03-17 10:45:30";
    //initialize blank date object
    std::tm date_obj = {};
    //wrap date in stream
    std::istringstream ss(date_str);
    //write date from stream to date object
    ss >> std::get_time(&date_obj, "%Y-%m-%d %H:%M:%S");
    //secondary string stream
    std::stringstream formatted_date_ss;
    //output date formatted
    formatted_date_ss << std::put_time(&date_obj, "%m/%d/%Y %H:%M:%S");
    std::string formatted_date = formatted_date_ss.str();
    //print formatted date
    std::cout << formatted_date << std::endl;


    return 0;
}
